src/static/js/modules/form-explainer.js

```diff
@@ -75,112 +75,182 @@ function updateStickiness() {
+    }, 600);
+    // After fading out the current page, fade in the new page.
+    var fadeInTimeout = window.setTimeout(function () {
+      $WRAPPER.find( '#explain_page-' + newCurrentPage ).fadeIn( 700 );
+      stickyHack();
+      window.clearTimeout( fadeInTimeout );
+    }, 1050);
+  }
+  // Update the pagination numbers.
+  $WRAPPER.find('.explain_pagination .pagination_current').text( newCurrentPage );
+  // Update the previous/next buttons if the new page is the first or last.
+  $('.explain_pagination .pagination_prev, .explain_pagination .pagination_next').removeClass('btn__disabled');
+  if ( newCurrentPage === 1 ) {
+    $WRAPPER.find('.explain_pagination .pagination_prev').addClass('btn__disabled');
+  } else if ( newCurrentPage === TOTAL ) {
+    $WRAPPER.find('.explain_pagination .pagination_next').addClass('btn__disabled');
+  }
+}
+
+/**
+ * Initialize a page.
  * @return {null}
  */
-function init() {
-  var $currentPage = getCurrentPage(),
+function initPage( id ) {
+  var $currentPage =      $WRAPPER.find( '#explain_page-' + id ),
       $imageMap =         $currentPage.find('.image-map'),
       $imageMapImage =    $currentPage.find('.image-map_image'),
       $imageMapWrapper =  $currentPage.find('.image-map_wrapper'),
       $terms =            $currentPage.find('.terms');
-  // Resize the image, terms and pagination columns
-  fitToWindow();
-  // Initiate a sticky image map only if the terms are taller than the window.
-  if ( $terms.height() > $WINDOW.height() ) {
+  if ( $WINDOW.width() >= 600 ) {
+    // Resize the image, terms and pagination columns
+    fitToWindow( id );
     // When the sticky plugin is applied to the image, it adds position fixed,
     // and the image's width is no longer constrained to its parent.
     // To fix this we will give it its own width that is equal to the parent.
     $imageMapImage.css( 'width', $imageMap.width() );
-    $imageMapWrapper.sticky();
-    $WINDOW.on( 'scroll', updateStickiness );
+    if ( $WINDOW.width() >= 600 ) {
+      $imageMapWrapper.sticky({ topSpacing: 30 });
+    }
   }
+  setCategoryPlaceholders( id );
+}
+
+/**
+ * Weird hack to get sticky() to update properly.
+ * We're basically jinggling the window to force a sticky() repaint.
+ * Sometimes it just needs a push I guess?
+ * @return {null}
+ */
+function stickyHack() {
+  $WINDOW.scrollTop( $WINDOW.scrollTop() + 1 );
+  $WINDOW.scrollTop( $WINDOW.scrollTop() - 1 );
 }
 
-// Do it!
+/**
+ * Set category placeholders
+ * @return {null}
+ */
+function setCategoryPlaceholders( id ) {
+  var $currentPage = $WRAPPER.find( '#explain_page-' + id ),
+      categories = [ 'checklist', 'alerts', 'definitions' ];
+  for ( var i = 0; i < categories.length; i++ ) {
+    var category = categories[i];
+    if ( $currentPage.find( '.expandable__form-explainer-' + category ).length === 0 ) {
+      placeholderHTML = '' +
+      '<div class="expandable expandable__padded expandable__form-explainer ' +
```

Comment:
They're used a lot in `check-rates.js.` I'm happy to go over how to get it started here.